contracts/executable/AxelarExecutable.sol

```diff
@@ -2,67 +2,8 @@
 
 pragma solidity ^0.8.0;
 
-import { IAxelarGateway } from '../interfaces/IAxelarGateway.sol';
-import { IAxelarExecutable } from '../interfaces/IAxelarExecutable.sol';
+import { AxelarGmpWithTokenExecutable } from './AxelarGmpWithTokenExecutable.sol';
 
-contract AxelarExecutable is IAxelarExecutable {
-    IAxelarGateway public immutable gateway;
-
-    constructor(address gateway_) {
-        if (gateway_ == address(0)) revert InvalidAddress();
-
-        gateway = IAxelarGateway(gateway_);
-    }
-
-    function execute(
-        bytes32 commandId,
-        string calldata sourceChain,
-        string calldata sourceAddress,
-        bytes calldata payload
-    ) external {
-        bytes32 payloadHash = keccak256(payload);
-
-        if (!gateway.validateContractCall(commandId, sourceChain, sourceAddress, payloadHash))
-            revert NotApprovedByGateway();
-
-        _execute(sourceChain, sourceAddress, payload);
-    }
-
-    function executeWithToken(
-        bytes32 commandId,
-        string calldata sourceChain,
-        string calldata sourceAddress,
-        bytes calldata payload,
-        string calldata tokenSymbol,
-        uint256 amount
-    ) external {
-        bytes32 payloadHash = keccak256(payload);
-
-        if (
-            !gateway.validateContractCallAndMint(
-                commandId,
-                sourceChain,
-                sourceAddress,
-                payloadHash,
-                tokenSymbol,
-                amount
-            )
-        ) revert NotApprovedByGateway();
-
-        _executeWithToken(sourceChain, sourceAddress, payload, tokenSymbol, amount);
-    }
-
-    function _execute(
-        string calldata sourceChain,
-        string calldata sourceAddress,
-        bytes calldata payload
-    ) internal virtual {}
-
-    function _executeWithToken(
-        string calldata sourceChain,
-        string calldata sourceAddress,
-        bytes calldata payload,
-        string calldata tokenSymbol,
-        uint256 amount
-    ) internal virtual {}
+contract AxelarExecutable is AxelarGmpWithTokenExecutable {
+    constructor(address gateway_) AxelarGmpWithTokenExecutable(gateway_) {}
```

Comment:
let's keep the original as-is, the new ones can also include `commandId` as a result?